static/js/exchange.js

```diff
@@ -49,7 +49,8 @@ updateExchange()
 const tableOptions = {
   dataSource: new UrlDataSource(exchangeUrl + '/bindings/source', { useQueryState: false }),
   pagination: true,
-  keyColumns: ['destination', 'properties_key']
+  keyColumns: ['destination', 'properties_key'],
+  search: true
```

Comment:
because it's the default? Or you're not allowed to search the exchange table anymore?